java-bootcamp-2018/src/com/globant/Singleton/DatabaseConnection.java

```diff
@@ -0,0 +1,20 @@
+package com.globant.Singleton;
+
+public class DatabaseConnection {
+
+	private DbConnection db;
+	static DatabaseConnection obj = new DatabaseConnection();
+

```

Comment:
Please implement the singleton with lazy inizialization, not at the start up of the app. You are also maintaining two different objects in the same singleton.

Also rename the package to lowercase. Packages are mostly one word and in lowercase.